chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -40,6 +40,16 @@ for router in $routers; do
     # delete old routers namespaces. Unassignment didn't delete the namespaces
     ssh $old_host ip netns delete qrouter-$router
 
+    # wait for router ports to migrate
+    ports=$(neutron router-port-list $router -f value -c id)
+    for port in $ports; do
+        port_host=$(neutron port-show $router -c binding:host_id -f value)
+        while [ $port_host != $(hostname) ]; do
```

Comment:
Can we avoid evaluating `$(hostname)` all the time? We should just do it once outside of all loops :-)